src/api/app/models/canned_response.rb

```diff
@@ -9,6 +9,12 @@ class CannedResponse < ApplicationRecord
   validates :title, length: { maximum: 255 }
   validates :content, length: { maximum: 65_535 }
 
+  enum kind: {
```

Comment:
I wouldn't do that if I keep the three values I have right now as `generic` is not a kind of decision:

```
0: generic
1: cleared
2: favor
```
because `generic` is not a kind of decision (maybe a kind of canned response). To be more precise with the other two, I could rename them as `cleared_decision` and `favor_decision` and humanize those values in the UI.

BTW, I don't discard refactoring all of this and make `canned_responses.kind` be nil by default (nil = "generic") so then I can define the enum for the `decision_kind` only. I'm going to have a look soonish to see if that's possible and doesn't overcomplicate things too much.

Reply:
I finally decided to go for

```
nil: generic
0: cleared
1: favor
```

Reply:
Why don't you use this list:
```
0: generic
1: cleared
2: favor
```
And then set a default of 0? 
To me it makes a bit more sense than having a nil as default but that nil means something (nil usually means "nothing")...

Reply:
@danidoni what you mention is what I had at the beginning. But semantically, the field represents a `decision kind ` and `generic` is not one of those.

Reply:
Alright. Let's go with this.